__docs__/wonder-blocks-form/text-field.stories.tsx

```diff
@@ -419,50 +423,145 @@ function ErrorRender() {
         }
     };
 
-    const handleFocus = () => {
-        setFocused(true);
-    };
-
-    const handleBlur = () => {
-        setFocused(false);
-    };
-
     return (
         <View>
             <TextField
                 id="tf-7"
                 type="email"
-                value={value}
                 placeholder="Email"
                 validate={validate}
                 onValidate={handleValidate}
-                onChange={handleChange}
                 onKeyDown={handleKeyDown}
-                onFocus={handleFocus}
-                onBlur={handleBlur}
+                {...args}
+                value={value}
+                onChange={handleChange}
             />
-            {!focused && errorMessage && (
+            {(errorMessage || args.error) && (
                 <View>
-                    <Strut size={spacing.xSmall_8} />
-                    <_Text style={styles.errorMessage}>{errorMessage}</_Text>
+                    <Strut size={spacing.xxSmall_6} />
+                    <LabelSmall style={styles.errorMessage}>
+                        {errorMessage || "Error from error prop"}
+                    </LabelSmall>
                 </View>
             )}
         </View>
     );
 }
 
+/**
+ * If the `error` prop is set to true, the TextField will have error styling and
+ * `aria-invalid` set to `true`.
+ *
+ * This is useful for scenarios where we want to show an error on a
+ * specific field after a form is submitted (server validation).
+ *
+ * Note: The `required` and `validate` props can also put the TextField in an
+ * error state.
+ */
+export const Error: StoryComponentType = {
+    render: ErrorRender,
+    args: {
+        error: true,
+        validate: undefined,
+    },
+};
+
 /**
  * If an input value fails validation, `TextField` will have error styling.
  *
+ * This is useful for scenarios where we want to show errors while a
+ * user is filling out a form (client validation).
+ *
  * Note that we will internally set the correct `aria-invalid` attribute to the
  * `input` element:
- * - aria-invalid="true" if there is an error message.
- * - aria-invalid="false" if there is no error message.
+ * - aria-invalid="true" if there is an error.
+ * - aria-invalid="false" if there is no error.
  */
-export const Error: StoryComponentType = {
+export const ErrorFromValidation: StoryComponentType = {
     render: ErrorRender,
 };
```

Comment:
same question here to understand if both stories need to include visual snapshots.